controllers/index.js

```diff
@@ -0,0 +1,6 @@
+module.exports.home = function (req, res, next) {
+    res.send("Welcome to Deskify");
+}
+module.exports.userRegistration = function (req, res, next) {
+    res.send("Welcome to User Registration");
+}
```

Comment:
```suggestion

```
I don't think this will be needed as the main page will be supplied by an html file in /public (on another branch)